android/app/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
```

Comment:
@cristipaval Would you mind checking these changes in this file? I was running into an error when building on android and these changes fixed it: https://github.com/Expensify/App/pull/14802#pullrequestreview-1305562512

Reply:
Hey, this is fine. The error occurs because a library added as dependency to the App has `android:supportsRtl="true"` in its `AndroidManifext.xml` file. Or maybe the library doesn't have that property set at all, which means that the compiler sets the default value which is `true`. This is in conflict with our App settings (`android:supportsRtl="false"`) and the line added (`tools:replace="android:supportsRtl"`) tells the compiler to set our App's value to that property to all libraries when there is a conflict. This is fine, because the setting disables support for languages which are written from right to left and we don't support those languages yet.